src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.test.tsx

```diff
@@ -31,20 +30,18 @@ import {
 } from 'tests/fixtures/entity-viewer/gene';
 
 jest.mock('../transcripts-list-item-info/TranscriptsListItemInfo', () => () => (
-  <div>TranscriptsListItemInfo</div>
+  <div className="transcriptsListItemInfo">TranscriptsListItemInfo</div>
```

Comment:
Here and elsewhere, why are we using `className` instead of `data-test-id`?

Reply:
I don't have a good explanation other than we used classes even before we started using data-test-ids. In our source code I try never to use classes for anything other than styling purposes, so I would be opposed to adding a class to an element in a component for the sole purpose of identifying this element in a test, and would use the data-test-id attribute instead. On the other hand, in tests, we already commonly use classes of various elements in a component to identify them in a test. I think we do it out of convenience and economy — we don't wan't to litter our source code with the data-test-id attributes. Now when it comes to which kind of identifier we should use in a mock, data-test-id would be cleaner, but ultimately it shouldn't really matter.

In a word: no reason, data-test-id probably slightly better semantically. I can change the ones I can find in this PR, but it's unlikely to improve anything 😃

Reply:
Thanks for the great explanation. I was curious about why using both of them instead of one, especially in mocked components like this one.

Reply:
I've replaced searches by className with searches by data-test-id for all mocked components I could find in this PR. Just for purity's sake 😃